package.json

```diff
@@ -15,7 +15,7 @@
     "lint": "vue-cli-service lint",
     "serve": "concurrently --names 'SERVER,CLIENT' --prefix-colors 'yellow,blue' --success \"first\" --kill-others 'yarn run serve:jupyterhub' 'yarn run serve:vue'",
     "serve:vue": "vue-cli-service serve --mode offline",
-    "serve:jupyterhub": "json-server --port 3000 ./src/services/mock/db.json",
+    "serve:jupyterhub": "nodemon src/services/mock/json-server.js",
```

Comment:
We are not running `json-server` any longer, but instead running `nodemon` to monitor for changes, and starting `json-server` programmatically. It's the documented way in `json-server` when you want to have dynamic responses — as is our case with the `/graphql` and the `/subscriptions` endpoints, which may return Tree data, or GScan data, or Dashboard, etc.